src/components/legend/attributes.js

```diff
@@ -106,10 +105,21 @@ module.exports = {
             'Determines the behavior on legend item double-click.',
             '*toggle* toggles the visibility of the item clicked on the graph.',
             '*toggleothers* makes the clicked item the sole visible item on the graph.',
-            '*false* disable legend item double-click interactions.'
+            '*false* disables legend item double-click interactions.'
+        ].join(' ')
+    },
+    groupclick: {
+        valType: 'enumerated',
+        values: ['toggleitem', 'togglegroup', false],
+        dflt: 'togglegroup',
+        editType: 'legend',
+        description: [
+            'Determines the behavior on legend group item click.',
+            '*toggleitem* toggles the visibility of the individual item clicked on the graph.',
+            '*togglegroup* toggles the visibility of all items in the same legendgroup as the item clicked on the graph.',
+            '*false* disables legend group click interactions.'
```

Comment:
What's the use case for `groupclick: false` that isn't covered by `itemclick: false`? I guess you have some traces in a group and others ungrouped, and you want to allow clicks on the ungrouped traces but not on the groups? That feels weird. Unless I'm missing something, can we remove `false` as an option here?

Reply:
I tend to agree that `false` is not a valid option here, and that `groupclick` should only determine what happens when you click on a grouped item AND `itemclick` is not `false`